index.js

```diff
@@ -24,10 +24,12 @@ module.exports = function buildAirbnbPreset(context, options) {
   var transpileTargets = (options && options.targets) ||
     buildTargets(options || {});
 
+  var debug = options.debug !== undefined ? options.debug : true;
```

Comment:
I've switched to false by default.

But I don't really get why should we use `!!` before `options.debug`. After all if type of options.debug is **boolean** then the value is (bool) `true` or (bool) `false`